graphql/validate.lua

```diff
@@ -262,6 +410,44 @@ local visitors = {
     rules = { rules.uniqueInputObjectFields }
   },
 
+  -- <inputObject>
+  --
+  -- {
+  --   kind = 'inputObject',
+  --   values = list of {
+  --     name = <string>,
+  --     value = <value>,
+  --   },
+  -- }
+  --[[
+  inputObject = {
```

Comment:
Probably full featured example would be more helpful here

Reply:
I would stick with just formal description for now.